apis_v1/views/views_campaign.py

```diff
@@ -89,6 +89,7 @@ def campaignx_retrieve_view(request):  # campaignRetrieve (CDN)
     voter_device_id = get_voter_device_id(request)  # We standardize how we take in the voter_device_id
     campaignx_we_vote_id = request.GET.get('campaignx_we_vote_id', '')
     hostname = request.GET.get('hostname', '')
+    print("\nhostname: " + hostname)              #*****EDIT*****#
```

Comment:
Thank you @nirvanabear for your check in. Please remove this print line.

Reply:
Oops! Print statement removed.